Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2021"
 name = "bzip2"
-version = "0.5.2"
+version = "0.5.3"
```

Comment:
is this change worth a major version bump? I'd say no, because the exported C symbols are an implementation detail. But technically it could break something